<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #17171B;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-250</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/background/ui/background/login.png);
background-repeat: no-repeat; /* Prevents the image from repeating */
background-color: none;
background-position: center; /* Centers the image */</string>
   </property>
   <widget class="QLineEdit" name="username_input">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>570</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #17171B;  /* Background color */
    border: 2px solid rgba(52, 51, 67, 255);  /* Default border color */
    border-radius: 8px;  /* Rounded corners */
    padding: 8px;  /* Padding inside the input box */
    color: white;  /* Text color */
    selection-background-color: rgba(52, 51, 67, 255); /* Selection background color */
    selection-color: blue;  /* Selected text color */
}

QLineEdit:focus {
    border: 2px solid rgba(251, 109, 169, 255);  /* Border color on focus (gradient2) */
    outline: none;  /* Remove default outline */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_title_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white;  /* Text color */
    font-size: 40px;  /* Font size */
	font-weight: bold;  /* Make the font bold */
	background-color: transparent;  /* Transparent background */
}
</string>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>640</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, 
        x2:1, y2:0, 
        stop:0 rgba(187, 63, 221, 255),  /* gradient1 */
        stop:0.5 rgba(251, 109, 169, 255),  /* gradient2 */
        stop:1 rgba(255, 159, 124, 255)   /* gradient3 */
    );
    color: white;  /* Text color */
    border: 2px solid rgba(52, 51, 67, 255);  /* Border color */
    border-radius: 10px;  /* Rounded corners */
    padding: 10px;  /* Padding around text */
}

QPushButton:hover {
    background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, 
        x2:1, y2:0, 
        stop:0 rgba(207, 83, 241, 255),  /* Slightly lighter gradient1 for hover */
        stop:0.5 rgba(255, 129, 189, 255),  /* Slightly lighter gradient2 for hover */
        stop:1 rgba(255, 179, 144, 255)   /* Slightly lighter gradient3 for hover */
    );
}

QPushButton:pressed {
    background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, 
        x2:1, y2:0, 
        stop:0 rgba(167, 43, 191, 255),  /* Darker gradient1 for pressed state */
        stop:0.5 rgba(231, 89, 149, 255),  /* Darker gradient2 for pressed state */
        stop:1 rgba(235, 139, 114, 255)   /* Darker gradient3 for pressed state */
    );
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>605</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: red;  /* Text color */
    font-size: 16px;  /* Font size */
	background-color: transparent;  /* Transparent background */
}
</string>
    </property>
    <property name="text">
     <string>Username is required</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../ui.qrc"/>
 </resources>
 <connections/>
</ui>
